client/app/account/settings/settings.html

```diff
@@ -1,54 +1,79 @@
 
 
 <div class="container">
+
   <div class="row">
     <div class="col-sm-12">
-      <h1>Change Password</h1>
+      <h3>Email Preferences</h3>
     </div>
     <div class="col-sm-12">
-      <form class="form" name="form" ng-submit="vm.changePassword(form)" novalidate>
+      <form class="form" name="emailPreferencesForm" ng-submit="updateEmailPreferences(emailPreferencesForm)" novalidate>
+        <div class="form-check">
+          <label class="form-check-label">
+            <input type="checkbox" class="form-check-input" ng-model="emailPreferences.unsubscribe">
+            Unsubscribe from emails
+          </label>
+        </div>
+        <h5>Event Reminder</h5>
+        <div ng-repeat='emailAhead in emailAheadOptions' class="form-check disabled">
+          <label class="form-check-label">
+            <input type="checkbox" class="form-check-input" ng-model="emailAhead.applied" ng-disabled="!emailPreferences.unsubscribe">
```

Comment:
This is confusing. The user should not have to click unsubscribe to change the event reminders. Instead they should be able to select any of them and click update (remove the Unsubscribe from emails checkbox).  There should also be a "Unsubscribe from all" button that deselects all and submits.